doc/classes/Color.xml

```diff
@@ -63,13 +63,13 @@
 			<argument index="3" name="a" type="float">
 			</argument>
 			<description>
-				Constructs a [Color] from an RGBA profile using values between 0 and 1.
+				Constructs a [Color] from RGBA values, typically between 0 and 1.
 				[codeblocks]
 				[gdscript]
-				var color = Color(0.2, 1.0, 0.7, 0.8) # Equivalent to RGBA(51, 255, 178, 204)
+				var color = Color(0.2, 1.0, 0.7, 0.8) # Similar to `Color8(51, 255, 178, 204)`
```

Comment:
I think the point of the `RGBA()` pseudo code was to show that the four parameters encode red, green, blue and optionally alpha in order. Not sure the change to `Color8()` preserves this meaning, though it makes it valid code and not pseudo code.

Reply:
Well looking at further examples, maybe not. It was just that whoever wrote the docs really preferred 8-bit I guess... So the change is fine

Reply:
The order of the parameters should already be conveyed via the constructor argument order above the code example, and it also shows up when writing the code in GDScript.